src/Chapter08.Tests/Listing08.03.ImplementingAnInterface.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter08.Listing08_03.Tests
+{
+    [TestClass]
+    public class ContactTests
+    {
+        [TestMethod]
+        public void CompareTo_ObjIsNull_Return1()
+        {
+            Contact contact = new Contact("Inigo Montoya");
+            
+            Assert.AreEqual(1, contact.CompareTo(null));
+        }
+
+        [TestMethod]
+        public void CompareTo_ObjLargerThanThis_ReturnNegative1()
+        {
+            Contact firstContact = new Contact("Inigo Montoya");
+            Contact secondContact = new Contact("Zero Zero");
+            
+            Assert.AreEqual(-1, firstContact.CompareTo(secondContact));
+        }
+
+        [TestMethod]
+        public void CompareTo_ObjLargerThanThis_Return1()
```

Comment:
This test and the one above appear to have the same condition name. One should be renamed